mavedb_mapping/blat_alignment.py

```diff
@@ -0,0 +1,369 @@
+from Bio import SearchIO
+import pandas as pd
+import subprocess
+
+from mavedb_mapping import qh
+from mavedb_mapping import path_to_hg38_file
+
+
+def get_gene_symb(dat:dict):
+    """
+    Obtains gene symbol using gene normalizer.
+
+    Parameters
+    ----------
+        dat: dict
+            Dictionary containing data required for mapping from MAVEDB scoreset.
+
+
+    Returns:
+    --------
+        str:
+            Gene symbol
+
+        If gene symbol cannot be extracted, returns None.
+
+    """
+    try:
+        uniprot = dat["uniprot_id"]
+        gsymb = qh.normalize(str(f"uniprot:{uniprot}")).gene_descriptor.label
+    except:
+        try:
+            target = dat["target"].split(" ")[0]
+            gsymb = qh.normalize(target).gene_descriptor.label
+        except:
+            return None
+    return gsymb
+
+
+def get_gene_data(gsymb: str)->tuple:
+    """
+    Searches through QueryHandler using obtained gene symbol.
+
+    Parameters
+    ----------
+        gsymb: str
+            Gene symbol
+
+    Returns:
+    --------
+        tuple:
+            source_dict: dictionary of sources with their index in temp
+            temp: QueryHandler search output
+    
+    """
+    if not gsymb:
+        return (None, None)
+    temp = qh.search(gsymb).source_matches
+    source_dict = {}
+    for i in range(len(temp)):
+        source_dict[temp[i].source] = i
+    return temp, source_dict
```

Comment:
Good variable naming is hard (much harder than the actual code writing part!). I'd recommend a more descriptive variable name than `temp`, e.g. maybe something like `source_matches`